great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -412,8 +412,11 @@ class MetricDomainTypes.
 
         # Checking if table or identity or other provided, column is not specified. If it is, warning the user
         elif domain_type == MetricDomainTypes.MULTICOLUMN:
-            if "columns" in compute_domain_kwargs:
-                accessor_domain_kwargs["columns"] = compute_domain_kwargs.pop("columns")
+            if "column_list" in compute_domain_kwargs:
+                # If column_list exists
```

Comment:
I do think that this code snippet is clear enough that this comment is probably unnecessary. That being said, I will always err on the side of overdocumentng things. Thoughts?